xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -331,6 +331,11 @@ void CRenderer::SetVideoRect(CRect &source, CRect &dest, CRect &view)
   m_rv = view;
 }
 
+void CRenderer::SetStereoMode(std::string &stereomode)
```

Comment:
If passing by reference it should be always by const reference, if it is for consumption only.
```suggestion
void CRenderer::SetStereoMode(const std::string& stereomode)
```